public/authorize.html

```diff
@@ -3,14 +3,14 @@
   <head>
     <meta charset="utf-8">
     <title>Authorize</title>
-    <link rel="stylesheet" href="./css/authorize.css">
+    <link rel="stylesheet" href="css/authorize.css">
```

Comment:
place the `/` at the beginning of the href to be sure to use the absolute path from the host, and not the relative from where you are.